source/elements/oneDNN/source/graph/ops/activation/ClampBackprop_1.rst

```diff
@@ -0,0 +1,69 @@
+.. SPDX-FileCopyrightText: 2020-2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+-------------
+ClampBackprop
```

Comment:
Do we want to align the naming here with API layer ([link](https://github.com/oneapi-src/oneAPI-spec/blob/bc6f418378501e1a62a55329f813ad72aa11053c/source/elements/oneDNN/include/dnnl_graph.hpp#L416)), for example `ClampBackprop` -> `ClampBackward`. Similar for the file name (`ClampBackprop_1.rst` -> `ClampBackward_1.rst`) and other backward ops.

Reply:
Yes we do. Thanks for catching those.